templates/shared_services/databricks-auth/porter.yaml

```diff
@@ -2,7 +2,7 @@
 schemaVersion: 1.0.0
 name: tre-shared-service-databricks-private-auth
 version: 0.1.1
```

Comment:
Does this version number not need to increase?

Reply:
Thanks.